### ISSUE TITLE
does not seem to work with "Automatically hide and show the menu bar"

### ISSUE THREAD

**Issue by visualex** [dexterleng/vimac#358]
if you enable Settings/General - "Automatically hide and show the menu bar" (which is a really useful feature to remove extra distractions) the shortcut keys appear where the menu bar should be, but typing them does not do what is expected: show bar and click the expected menu Item.

**Comment by dexterleng** [comment]
Interesting - didn't know you could hide the menu bar.

**Comment by dexterleng** [comment]
A simple solution is to not show hints for menu bar items if they are not visible.

To allow the user to click on menu bar items when it is hidden by default - The problem is similar to supporting clicking on Dock items for people who choose to hide the Dock - If I show the menu bar when Hint-mode is activated, it could block hints on the top of the display.

**Comment by visualex** [comment]
yeah there is an option to set focus on the menu bar in keyboard shortcuts, so I suppose covering both with/without it would be a fix. Thanks!!

**Comment by albert-ying** [comment]
If I remember correctly, few versions ago the hidden menu bar will be automatically revealed after toggling the hint. But now this behavior is gone some how, therefore it would be nice to add an option for not showing hints for menu bar items.
